src/binarySearch.js

```diff
@@ -0,0 +1,22 @@
+const BinarySearch = (inputArray, searchValue) => {
```

Comment:
Hello! Very clean code, easy to follow step-by-step. One area of improvement I feel could be comments :) Clean code combined with comments will make this super code!